src/main/java/nl/stil4m/mollie/domain/Payment.java

```diff
@@ -36,29 +36,30 @@
-                   @JsonProperty("mode") String mode,
-                   @JsonProperty("customerId") String customerId,
-                   @JsonProperty("recurringType") String recurringType,
-                   @JsonProperty("createdDatetime") Date createdDatetime,
-                   @JsonProperty("status") String status,
-                   @JsonProperty("paidDatetime") String paidDatetime,
-                   @JsonProperty("cancelledDatetime") String cancelledDatetime,
-                   @JsonProperty("expiredDatetime") String expiredDatetime,
-                   @JsonProperty("amount") Double amount,
-                   @JsonProperty("amountRefunded") Double amountRefunded,
-                   @JsonProperty("amountRemaining") Double amountRemaining,
-                   @JsonProperty("description") String description,
-                   @JsonProperty("method") String method,
-                   @JsonProperty("details") Map<String, Object> details,
-                   @JsonProperty("links") Links links,
-                   @JsonProperty("metadata") Map<String, Object> metadata,
-                   @JsonProperty("locale") String locale,
-                   @JsonProperty("countryCode") String countryCode,
-                   @JsonProperty("expiryPeriod") String expiryPeriod,
-                   @JsonProperty("mandateId") Optional<String> mandateId,
-                   @JsonProperty("canBeCancelled") Optional<Boolean> canBeCancelled) {
+            @JsonProperty("id") String id,
+            @JsonProperty("profileId") String profileId,
+            @JsonProperty("mode") String mode,
+            @JsonProperty("customerId") String customerId,
+            @JsonProperty("recurringType") String recurringType,
+            @JsonProperty("createdDatetime") Date createdDatetime,
+            @JsonProperty("status") String status,
+            @JsonProperty("paidDatetime") Date paidDatetime,
+            @JsonProperty("cancelledDatetime") Date cancelledDatetime,
+            @JsonProperty("expiredDatetime") Date expiredDatetime,
+            @JsonProperty("amount") Double amount,
+            @JsonProperty("amountRefunded") Double amountRefunded,
+            @JsonProperty("amountRemaining") Double amountRemaining,
+            @JsonProperty("description") String description,
+            @JsonProperty("method") String method,
+            @JsonProperty("details") Map<String, Object> details,
+            @JsonProperty("links") Links links,
+            @JsonProperty("metadata") Map<String, Object> metadata,
+            @JsonProperty("locale") String locale,
+            @JsonProperty("countryCode") String countryCode,
+            @JsonProperty("expiryPeriod") String expiryPeriod,
+            @JsonProperty("mandateId") Optional<String> mandateId,
+            @JsonProperty("canBeCancelled") Optional<Boolean> canBeCancelled,
+            @JsonProperty("failedDatetime") Date failedDatetime) {
```

Comment:
I only sea `failedAt` in the API (https://docs.mollie.com/reference/v2/payments-api/get-payment). Where does this come from?

Reply:
you're looking at v2, this version recently released
check https://docs.mollie.com/reference/v1/payments-api/get-payment